atmos_model.F90

```diff
@@ -157,9 +160,8 @@ module atmos_model_mod
 !logical :: debug        = .true.
 logical :: sync         = .false.
 integer, parameter     :: maxhr = 4096
```

Comment:
`maxhr` is not used anymore. Should be deleted.

Reply:
done